phylo2owl.py

```diff
@@ -28,6 +32,13 @@
     help='Phylogeny file to parse'
 )
 cmdline_parser.add_argument(
+    '-t', '--type', dest='input_type', nargs='?',
```

Comment:
I'm wondering whether we shouldn't better name this `-f` and `--format`, respectively; my feeling is that's much more common and thus more easily remembered?